release/rllib_tests/autoscaler-cluster.yaml

```diff
@@ -0,0 +1,59 @@
+cluster_name: ray-rllib-regression-tests
+
+min_workers: 0
+max_workers: 0
+
+docker:
+    image: anyscale/ray-ml:latest-gpu
+    container_name: ray_container
+    pull_before_run: True
+    run_options: ["--ulimit nofile=1045876"]
+
+# Cloud-provider specific configuration.
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+    cache_stopped_nodes: False
+
+# How Ray will authenticate with newly launched nodes.
+auth:
+    ssh_user: ubuntu
+
+head_node:
+    InstanceType: p3.16xlarge
```

Comment:
IIRC we used to run these tests on cheaper/smaller instances? Does it matter for benchmarking purposes? If not then lgtm.